src/components/SearchResults/SearchResults.tsx

```diff
@@ -13,12 +13,23 @@ const SearchResults: React.FC<RouteComponentProps> = (props) => {
   let query: unknown | any = props.location.state
 
   useEffect(() => {
+    const filterMatches = (data:SearchResponse) => {
```

Comment:
Thank you for adding this filter method! Can't wait for that endpoint to be up and working so we can see some of the site in action. :)